pom.xml

```diff
@@ -0,0 +1,218 @@
+    <osgi.import.packages>
+      org.fcrepo.auth.roles.common,
+      org.fcrepo.kernel.api.*,
+      org.fcrepo.http.commons.*,
+      org.fcrepo.kernel.modeshape.*,
+
+      javax.jcr.*,
+      javax.annotation,
+      javax.xml.*,
+      javax.inject,
+      javax.servlet.http,
+
+      com.google.common.*,
+      com.hp.hpl.jena.*,
+
+      org.apache.commons.io
+      org.modeshape.jcr.api.*,
+      org.springframework.beans.*,
+      org.springframework.stereotype,
+      org.w3c.dom.*,
+
+      *
+    </osgi.import.packages>
+    <osgi.export.packages>
+      org.fcrepo.auth.webac;version=${project.version}
+    </osgi.export.packages>
+  </properties>
+
+  <repositories>
+    <repository>
+      <id>snapshots-repo</id>
+      <url>http://oss.sonatype.org/content/repositories/snapshots</url>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
+  </repositories>
+
+  <scm>
+    <connection>scm:git:git://github.com/fcrepo4/${project.artifactId}.git
+    </connection>
+    <developerConnection>scm:git:git@github.com:fcrepo4/${project.artifactId}.git
+    </developerConnection>
+    <url>https://github.com/fcrepo4/fcrepo-module-auth-webac</url>
+    <tag>HEAD</tag>
+  </scm>
+
+
+  <dependencies>
+
+    <!-- test gear -->
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-core</artifactId>
+        <scope>test</scope>
+      </dependency>
+
+    <dependency>
+      <groupId>org.glassfish.grizzly</groupId>
+      <artifactId>grizzly-http-server</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.grizzly</groupId>
+      <artifactId>grizzly-http-servlet</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.test-framework.providers</groupId>
+      <artifactId>jersey-test-framework-provider-grizzly2</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.ext</groupId>
+      <artifactId>jersey-spring3</artifactId>
+      <scope>test</scope>
+      <version>${jersey.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.core</groupId>
+      <artifactId>jersey-common</artifactId>
+      <version>${jersey.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-cachestore-leveldb</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
The infinispan-cachestore-leveldb dependency is causing a dependency conflict and can be safely removed